classes/plugin.class.php

```diff
@@ -82,7 +82,7 @@ function __construct() {
 			// add_action ( 'woocommerce_before_calculate_totals', 'ppom_before_calculate_totals', 999, 1 );
 
 			// Above hook has issue so reverting back to old hook for prices
-			add_filter( 'woocommerce_get_cart_item_from_session', 'ppom_price_controller', 10, 2 );
+			// add_filter( 'woocommerce_get_cart_item_from_session', 'ppom_price_controller', 10, 2 );
 
```

Comment:
We should preserve that's hugely important hook. That changes breaks price calculation feature almost totally. Of course that fixes price matrix but breaks lots of fields.

Just two example:
- Sample from Radio Field: https://vertis.d.pr/i/qoIYdj - Cart: https://vertis.d.pr/i/11c2Pe (the change is causing PPOM prices are passing into cart as 0 instead of 200$)
- Sample from Select Field: Product: https://vertis.d.pr/i/VTGuel  - Cart: https://vertis.d.pr/i/fk3rbM (the change is causing PPOM prices are passing into cart as 0)